azure-spring-boot/src/main/resources/serviceEndpoints.properties

```diff
@@ -2,8 +2,17 @@ azure.service.endpoints.cn.aadSigninUri=https://login.partner.microsoftonline.cn
 azure.service.endpoints.cn.aadGraphApiUri=https://graph.chinacloudapi.cn/
 azure.service.endpoints.cn.aadKeyDiscoveryUri=https://login.partner.microsoftonline.cn/common/discovery/keys
 azure.service.endpoints.cn.aadMembershipRestUri=https://graph.chinacloudapi.cn/me/memberOf?api-version=1.6
+azure.service.endpoints.cn-microsoftgraph.aadSigninUri=https://login.partner.microsoftonline.cn/
+azure.service.endpoints.cn-microsoftgraph.aadGraphApiUri=https://microsoftgraph.chinacloudapi.cn/
+azure.service.endpoints.cn-microsoftgraph.aadKeyDiscoveryUri=https://login.partner.microsoftonline.cn/common/discovery/keys
+azure.service.endpoints.cn-microsoftgraph.aadMembershipRestUri=https://microsoftgraph.chinacloudapi.cn/v1.0/me/memberOf
 azure.service.endpoints.global.aadSigninUri=https://login.microsoftonline.com/
```

Comment:
I think you should add some comments here to explain xx-microsoftgraph is associated with v2 version or change the properties' keys to more obvious keys, just like: azure.service.endpoints.cn-v2-graph.aadSigninUri